nf_core/lint.py

```diff
@@ -51,15 +51,20 @@ def lint_pipeline(self):
         Raises:
             If a critical problem is found, an AssertionError is raised.
         """
-        funcnames = [
+        normalchecks = [
             'check_files_exist',
             'check_licence',
             'check_docker',
             'check_config_vars',
             'check_ci_config',
             'check_readme'
         ]
-        with click.progressbar(funcnames, label='Running pipeline tests') as fnames:
+        releasechecks = [
+            'check_version_consistency'
+        ]
+        if os.environ.get('TRAVIS_TAG'): release=True
```

Comment:
Not `TRAVIS_TAG` - I think we said that it should be master branch only. So `TRAVIS_BRANCH` equals to `master`...

Reply:
Also, I think I'd prefer to have this in the click logic below, seems cleaner..

Reply:
I think I got you wrong the first time, so always make the checks, when on master or by flag, additionally check TAG if present :) had a deadlock in my brain.